--- a/CPAN/XML/Simple.pm	2025-12-26 21:25:47.032338139 +0100
+++ b/CPAN/XML/Simple.pm	2025-12-26 21:27:13.751247935 +0100
@@ -333,7 +333,7 @@
     # $tree = $xp->parsefile($filename);  # Changed due to prob w/mod_perl
     local(*XML_FILE);
     open(XML_FILE, '<', $filename) || croak qq($filename - $!);
-    $tree = $xp->parse(*XML_FILE);
+    $tree = $xp->parse(\*XML_FILE);
     close(XML_FILE);
   }
   else {
