--- a/CPAN/XML/Parser/Simple.pm	2025-01-05 03:27:46.708616463 +0100
+++ b/CPAN/XML/Parser/Simple.pm	2025-01-05 03:29:12.227541345 +0100
@@ -333,7 +333,7 @@
     # $tree = $xp->parsefile($filename);  # Changed due to prob w/mod_perl
     local(*XML_FILE);
     open(XML_FILE, '<', $filename) || croak qq($filename - $!);
-    $tree = $xp->parse(*XML_FILE);
+    $tree = $xp->parse(\*XML_FILE);
     close(XML_FILE);
   }
   else {
--- a/Slim/Control/Request.pm	2025-01-05 03:31:50.135556172 +0100
+++ b/Slim/Control/Request.pm	2025-01-05 03:32:59.984678050 +0100
@@ -1530,7 +1530,8 @@
 
 # get a parameter by name
 sub getParam {
-	return $_[0]->{'_params'}->{ $_[1] };
+	my $r = $_[0]->{'_params'}->{ $_[1] };
+	return ($r ne '') ? $r : undef;
 }
 
 # delete a parameter by name
--- a/Slim/Music/Info.pm	2025-01-05 03:34:45.643349741 +0100
+++ b/Slim/Music/Info.pm	2025-01-05 03:35:58.452434406 +0100
@@ -808,10 +808,10 @@
 	}
 
 	if ($client) {
-		$client->musicInfoTextCache($cache);
+#		$client->musicInfoTextCache($cache);
 	}
 	else {
-		$musicInfoTextCache = $cache;
+#		$musicInfoTextCache = $cache;
 	}
 
 	return $text;
