diff -Naur a/Slim/Player/Playlist.pm b/Slim/Player/Playlist.pm
--- a/Slim/Player/Playlist.pm	2024-04-01 11:25:23.354413622 +0200
+++ b/Slim/Player/Playlist.pm	2024-04-01 11:26:54.504414548 +0200
@@ -1219,7 +1219,7 @@
 	$id =~ s/://g;
 
 	return Slim::Utils::Misc::fileURLFromPath(
-		catfile(scalar Slim::Utils::OSDetect::dirsFor('prefs'), "clientplaylist_$id.m3u")
+		catfile(scalar Slim::Utils::OSDetect::dirsFor('ClientPlaylists'), "clientplaylist_$id.m3u")
 	);
 }
 
