diff -Naur a/CPAN/XML/Parser/Expat.pm b/CPAN/XML/Parser/Expat.pm
--- a/CPAN/XML/Parser/Expat.pm	2024-04-01 11:06:27.084402075 +0200
+++ b/CPAN/XML/Parser/Expat.pm	2024-04-01 11:08:18.814403211 +0200
@@ -490,7 +490,7 @@
   local(*FILE);
   open(FILE, $_[0]) or  croak "Couldn't open $_[0]:\n$!";
   binmode(FILE);
-  my $ret = $self->parse(*FILE);
+  my $ret = $self->parse(\*FILE);
   close(FILE);
   $ret;
 }
@@ -676,7 +676,7 @@
                       'End'   => \&eh,
                       'Char'  => \&ch);
  open(FOO, '<', 'info.xml') or die "Couldn't open";
- $parser->parse(*FOO);
+ $parser->parse(\*FOO);
  close(FOO);
  # $parser->parse('<foo id="me"> here <em>we</em> go </foo>');
 
diff -Naur a/CPAN/XML/Simple.pm b/CPAN/XML/Simple.pm
--- a/CPAN/XML/Simple.pm	2024-04-01 11:08:59.604403625 +0200
+++ b/CPAN/XML/Simple.pm	2024-04-01 11:09:28.514403919 +0200
@@ -333,7 +333,7 @@
     # $tree = $xp->parsefile($filename);  # Changed due to prob w/mod_perl
     local(*XML_FILE);
     open(XML_FILE, '<', $filename) || croak qq($filename - $!);
-    $tree = $xp->parse(*XML_FILE);
+    $tree = $xp->parse(\*XML_FILE);
     close(XML_FILE);
   }
   else {
diff -Naur a/Slim/Control/Request.pm b/Slim/Control/Request.pm
--- a/Slim/Control/Request.pm	2024-04-01 11:10:33.224404577 +0200
+++ b/Slim/Control/Request.pm	2024-04-01 11:11:39.524405250 +0200
@@ -1527,7 +1527,8 @@
 
 # get a parameter by name
 sub getParam {
-	return $_[0]->{'_params'}->{ $_[1] };
+	my $r = $_[0]->{'_params'}->{ $_[1] };
+	return ($r ne '') ? $r : undef;
 }
 
 # delete a parameter by name
diff -Naur a/Slim/Music/Info.pm b/Slim/Music/Info.pm
--- a/Slim/Music/Info.pm	2024-04-01 11:12:30.914405773 +0200
+++ b/Slim/Music/Info.pm	2024-04-01 11:14:11.064406790 +0200
@@ -807,10 +807,10 @@
 	}
 
 	if ($client) {
-		$client->musicInfoTextCache($cache);
+#		$client->musicInfoTextCache($cache);
 	}
 	else {
-		$musicInfoTextCache = $cache;
+#		$musicInfoTextCache = $cache;
 	}
 
 	return $text;
