diff -Naur a/Slim/Plugin/UPnP/Events.pm b/Slim/Plugin/UPnP/Events.pm
--- a/Slim/Plugin/UPnP/Events.pm	2024-04-01 12:27:55.720272721 +0200
+++ b/Slim/Plugin/UPnP/Events.pm	2024-04-01 12:29:44.348907074 +0200
@@ -13,7 +13,6 @@
 use HTTP::Date;
 use URI;
 use URI::QueryParam;
-use UUID::Tiny ();
 
 use Slim::Networking::Async;
 use Slim::Networking::Select;
@@ -235,7 +234,9 @@
 		$service =~ s{/}{::}g;
 		my $serviceClass = "Slim::Plugin::UPnP::$service";
 		
-		$uuid = uc( UUID::Tiny::create_UUID_as_string( UUID::Tiny::UUID_V4() ) );
+		require Data::UUID;
+		my $ug = new Data:UUID;
+		$uuid = uc $ug->create_str();
 		
 		$SUBS{ $uuid } = {
 			active    => 0, # Sub is not active until we send it to the subscriber
@@ -458,4 +459,4 @@
 	}
 }
 
-1;
\ Kein Zeilenumbruch am Dateiende.
+1;
diff -Naur a/slimserver.pl b/slimserver.pl
--- a/slimserver.pl	2024-04-01 12:30:27.688362223 +0200
+++ b/slimserver.pl	2024-04-01 12:31:23.677658342 +0200
@@ -477,8 +477,9 @@
 
 	# Generate a UUID for this SC instance on first-run
 	if ( !$prefs->get('server_uuid') ) {
-		require UUID::Tiny;
-		$prefs->set( server_uuid => UUID::Tiny::create_UUID_as_string( UUID::Tiny::UUID_V4() ) );
+		require Data::UUID;
+		my $ug = new Data::UUID;
+		$prefs->set( server_uuid => lc $ug->create_str() );
 	}
 
 	main::INFOLOG && $log->info("Server binary search path init...");
